engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -278,204 +278,210 @@ private void initRenderGraph() {
         renderPipelineTaskList = renderTaskListGenerator.generateFrom(orderedNodes);
     }
 
-    private void addShadowMapNodes(RenderGraph renderGraph) {
+    private void addShadowMapNodes(RenderGraph rGraph) {
```

Comment:
I *definitely* don't like this change.

Rather than rename all variables to rGraph, you can simply remove them from the parameters.

Reply:
@NicholasBatesNZ: what's the feedback checkstyle gives you regarding this variable?

A general rule of thumb in programming nowadays is to -not- abbreviate. It made sense once when we had to type everything, but nowadays with autocompletion and easy replacement available through the IDE softwares it no longer apply and clarity trumps brevity.

Reply:
Unless I'm mistaken, Checkstyle was complaining about variable hiding.

Reply:
Yes, it was variable hiding. I was not entirely sure what to do about it so I'll remove the parameter entirely as you suggest